packages/components-native/src/Typography/Typography.style.ts

```diff
@@ -32,6 +32,26 @@ const deviceFonts = {
     fontFamily: "inter-extrabold",
   },
 
+  baseItalicRegular: {
+    fontFamily: "inter-italic",
+  },
+
+  baseItalicMedium: {
+    fontFamily: "inter-italic-medium",
+  },
+
+  baseItalicBold: {
+    fontFamily: "inter-italic-bold",
+  },
+
+  baseItalicSemiBold: {
+    fontFamily: "inter-italic-semibold",
+  },
+
+  baseItalicExtraBold: {
+    fontFamily: "inter-italic-extrabold",
+  },
+
```

Comment:
All these new fonts coincide directly with fonts added to Jobber mobile [here](https://github.com/GetJobber/jobber-mobile/pull/9490/files)